### ISSUE TITLE
Aquamarine 0.3.x is broken

### ISSUE THREAD

**Issue by dashdeckers** [mersinvald/aquamarine#44]
When I use `aquamarine = "0.3.2"` the Mermaid diagram does not render and instead shows this error:

<img width="346" alt="image" src="https://github.com/mersinvald/aquamarine/assets/17221921/a438a444-579f-4c51-a59f-22fd05ae4379">

You can also see this on the [demo crate](https://docs.rs/aquamarine-demo-crate/latest/aquamarine_demo_crate/fn.example.html) when you go to the latest version. The demo crate docs.rs links for versions 0.2.0 up to and including 0.3.0 are broken (build failed), but when testing in my own repo I can say that 0.3.0 is also broken but 0.2.2 works fine.

**Comment by mersinvald** [comment]
It looks like mermaidjs update doesn't allow dots inside inline strings anymore:


This doesn't work
```rust
/// ```mermaid
/// graph LR
///     s([Source]) --> a[[aquamarine]]
///     r[[rustdoc]] --> f([Docs w/ Mermaid!])
///     subgraph rustc[Rust Compiler]
///     a -. inject mermaid.js .-> r
///     end
/// ```
///
```
```
ERROR :  Error parsing Error: Lexical error on line 5. Unrecognized text.
... a -. inject mermaid.js .-> r    end
```


This works
```rust
/// ```mermaid
/// graph LR
///     s([Source]) --> a[[aquamarine]]
///     r[[rustdoc]] --> f([Docs w/ Mermaid!])
///     subgraph rustc[Rust Compiler]
///     a -. "inject mermaid.js" .-> r
///     end
/// ```
///
```

I'll update the demo examples with a fix, but otherwise there doesn't seem to be a bug to fix, it's just the mermaid definitions that are broken

**Comment by mersinvald** [comment]
Fixed in 0.3.3
